fast/stages/03-gke-multitenant/prod/main.tf

```diff
@@ -73,8 +79,14 @@ module "gke-dataset-resource-usage" {
   friendly_name = "GKE resource usage."
 }
 
-resource "google_project_iam_member" "host_project_bindings" {
-  for_each = { for i, v in local.host_project_bindings : i => v }
+resource "google_project_iam_member" "host_project_gke_robot_bindings" {
```

Comment:
why split these in two? It easier to manage the depends_on that you need on the cluster if you have a single resource.